packages/terra-collapsible-menu-view/src/CollapsibleMenuViewToggle.jsx

```diff
@@ -1,7 +1,7 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
-import Control from 'terra-form/lib/Control';
+import Control from 'terra-form-checkbox';
```

Comment:
Maybe it's better to call it `Checkbox` here instead of `Control`?

Reply:
updated : 8e45afbd85e04adc7a369423f1105915413c71fc